.env.example

```diff
@@ -1,25 +1,19 @@
-# ${WORDPRESS_DB_NAME} and ${WORDPRESS_DB_PASSWORD} are passed from the environment variables
```

Comment:
This was removed to squash the spurious WARNings generated by docker-compose when running containers that don't have these variables set in the compose file.